lib/install/rust.js

```diff
@@ -0,0 +1,424 @@
+// System Objects
+var path = require('path');
+var spawn = require('child_process').spawn;
+var Transform = require('stream').Transform;
+var zlib = require('zlib');
+
+// Third Party Dependencies
+var blocks = require('block-stream2');
+var bz2 = require('unbzip2-stream');
+var createHash = require('sha.js');
+var fs = require('fs-extra');
+var fsTemp = require('fs-temp');
+var osenv = require('osenv');
+var Progress = require('progress');
+var request = require('request');
+var tar = require('tar-fs');
+
+// Internal
+var log = require('../log');
+
+function getPlatform() {
+  switch (process.platform) {
+    case 'darwin':
+      return 'macos';
+    case 'linux':
+      return 'linux';
+    default:
+      throw new Error('Your platform is not yet supported for cross-compilation.');
+  }
+}
+
+var SDK_PATHS = {
+  sdk: path.join(osenv.home(), '.tessel/sdk'),
+  rustlib: path.join(osenv.home(), '.tessel/rust'),
+};
+
+var SDK_URLS = {
+  macos: 'https://builds.tessel.io/t2/sdk/t2-sdk-macos-x86_64.tar.bz2',
```

Comment:
Lines 23-26 are unnecessary if you just rename "macos" => "darwin", otherwise it's unnecessary complexity